language/src/rustspec.rs

```diff
@@ -384,6 +384,7 @@ pub enum Pattern {
 
 #[derive(Clone, Serialize, Debug)]
 pub struct MutatedInfo {
+    pub is_option: bool,
```

Comment:
Rather than this boolean, I think it would be cleaner if we had :

```
enum ResultOrOption {
  Result,
  Option,
  Other
}
```

Reply:
I was thinking the same when defining it, but thought it could just be refactored when needed.